src/styles/_academy.scss

```diff
@@ -200,34 +200,71 @@
    */
   .ag-body-container {
     display: block;
+    /* Override ag-grid style to suppress unnecessary vertical scrollbar (bugfix) */
+    margin-bottom: 0;
   }
-  .ag-grid-export-button {
-    margin-top: 20px;
+
+  .pt-icon {
+    /* Vertically centre appearance of blueprintjs icons in ag-grid */
+    margin-top: 6px;
+  }
+
+  .ag-paging-panel {
+    border-top: none;
+    font-weight: bold;
+    padding: 10px;
+  }
+
+  :focus {
+    /* Disable rendering blue border when a sub-component in grid is in focus */
+    outline: none;
   }
 }
 
-.ag-header-cell-label {
-  justify-content: center;
-  font-weight: bold;
-  padding-left: 15px;
+.ag-grid-controls {
+  display: flex;
+  justify-content: space-between;
+
+  .pt-icon {
```

Comment:
We just bumped to blueprint 3. Use `.#{$ns}.icon`

Reply:
Have edited the stylesheet for this; thanks!